### PR TITLE
First step for Python 3 support

### PR DESCRIPTION
**Pull request by micheles**
Fixed all tests and doctest of risklib and commonlib. I am including the ElementTree code from the Python 3.3 standard library, until a better solution is found.
Nothing was broken: https://ci.openquake.org/job/zdevel_oq-engine/1989/

TODO: test this on Windows, run for instance DisaggXMLWriterTestCase, the change 'w' -> 'wb' may have a bad effect on the generated XML.

### CODE REVIEW

File: openquake/commonlib/tests/readinput_test.py [hunk 1]
```diff
@@ -783,6 +783,7 @@ def test(self):
         self.assertEqual(sorted(oqparam.hazard_imtls.items()),
                          [('PGA', [0.005, 0.007, 0.0137, 0.0337]),
                           ('SA(0.025)', [0.005, 0.007, 0.0137])])
+        print(sitecol.mesh.lons, sitecol.mesh.lats)
```
Comment (ptormene) [inline openquake/commonlib/tests/readinput_test.py]: Is it a debug print or is it useful in general?

Comment (micheles) [inline openquake/commonlib/tests/readinput_test.py]: It is a debug print that I forgot to remove

**Comment by ptormene** [comment]
LGTM

**Comment by daniviga** [comment]
LGTM

**Pull request merged**